packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1100,6 +1110,10 @@ function renderRoot(
     // There was a fatal error.
     if (__DEV__) {
       resetStackAfterFatalErrorInDev();
+
+      // Reset the DEV fiber stack in case we're profiling roots.
+      // (We do this for profiling bulds when DevTools is detected.)
+      resetActualRenderTimerStackAfterFatalErrorInDev();
```

Comment:
This clears out the host root from the profiler timer's DEV-only `fiberStack` in the event of an error (which prevents us from incorrectly warning on the next render).